jdaviz/configs/specviz/plugins/line_analysis/tests/test_line_analysis.py

```diff
@@ -400,4 +400,4 @@ def test_subset_changed(specviz_helper, spectrum1d):
     specviz_helper.app.state.drawer = True
 
     # Values have not yet been validated
-    np.testing.assert_allclose(float(plugin.results[0]['result']), 2.153181e-13, atol=1e-15)
+    np.testing.assert_allclose(float(plugin.results[0]['result']), 7.570859e-14, atol=1e-13)
```

Comment:
I had to change this test back and forth between `7.5...e-14` and `2.15...e-13` so I just changed the tolerance, hopefully that is ok.

Reply:
why did these change though? Because the boundaries of the subset are handled differently now?

Reply:
Exactly.

Reply:
If line analysis uses `get_subset_from_viewer` to retrieve the subsets then it is expected for these results to be different. As for version change, I can see the case for it but currently subset bounds (when using `get_subset_from_viewer`) are bugged. The method returns the bounds based on the closest spectral axis value rather than the bounds the user set. I could see the argument for bug release or minor version bump, but either way this is the correct behavior for subset bounds according to @camipacifici .